bindings/matrix-sdk-ffi/src/session_verification.rs

```diff
@@ -60,80 +73,130 @@ pub struct SessionVerificationController {
 
 #[matrix_sdk_ffi_macros::export]
 impl SessionVerificationController {
-    pub async fn is_verified(&self) -> Result<bool, ClientError> {
-        let device =
-            self.encryption.get_own_device().await?.context("Our own device is missing")?;
+    pub fn set_delegate(&self, delegate: Option<Box<dyn SessionVerificationControllerDelegate>>) {
+        *self.delegate.write().unwrap() = delegate;
+    }
+
+    /// Set this particular request as the currently active one and register for
+    /// events pertaining it.
+    /// * `sender_id` - The user requesting verification.
+    /// * `flow_id` - - The ID that uniquely identifies the verification flow.
+    pub async fn acknowledge_verification_request(
+        &self,
+        sender_id: String,
+        flow_id: String,
```

Comment:
Just a quick question regarding API usage: the `flow_id` is not enough here? We need to also provide the `sender_id`? (I believe that those data are coming from `SessionVerificationRequestDetails`).

Reply:
That's a good question, one I had myself. It's based on the underlying `OlmMachine::get_verification_request` which requires both, so there must be a good reason or it might be just because (maybe indexing by user ID so we can cancel existing requests for the same user https://github.com/matrix-org/matrix-rust-sdk/blob/main/crates/matrix-sdk-crypto/src/verification/machine.rs#L168-L169)

Reply:
OK, thanks. That's fine, I was just wondering :)